src/config.ts

```diff
@@ -32,15 +32,36 @@ export interface Config {
 const DEFAULTS: Config = {
   output: 'dist',
   pluginsDirectory: path.join(os.homedir(), '.cdk8s', 'plugins'),
+  imports: ['k8s'],
 };
 
-export function readConfigSync(): Config {
+export function readConfigSync(filePath?: string): Config {
   let config: Config = DEFAULTS;
-  if (fs.existsSync(CONFIG_FILE)) {
+  const fullFilePath = filePath ? path.join(filePath, CONFIG_FILE) : CONFIG_FILE;
+  if (fs.existsSync(fullFilePath)) {
+    const parsedYaml: Config = yaml.parse(fs.readFileSync(fullFilePath, 'utf-8'));
     config = {
       ...config,
-      ...yaml.parse(fs.readFileSync(CONFIG_FILE, 'utf-8')),
+      ...parsedYaml,
     };
   }
   return config;
+}
+
+export async function addImportToConfig(source: string, filePath?: string) {
+  const fullFilePath = filePath ? path.join(filePath, CONFIG_FILE) : CONFIG_FILE;
+  let curConfig = readConfigSync(filePath);
+
+  if (!curConfig.imports?.includes(source)) {
+    const importsList = curConfig.imports ?? [];
+    importsList.push(source);
+    let config = {
+      language: curConfig.language,
+      app: curConfig.app,
+      imports: importsList,
+      output: curConfig.output ?? DEFAULTS.output,
+      pluginsDirectory: curConfig.pluginsDirectory ?? DEFAULTS.pluginsDirectory,
```

Comment:
This is problematic because it would create a file with, for example:

```yaml
pluginsDirectory: /Users/epolon/.cdk8s/plugins
```

Notice the `/Users/epolon/` now makes this file non portable - for example to run in CI builds...